### PR TITLE
[stable/nginx-ingress] Update nginx-ingress README for deprecation

### PR DESCRIPTION
**Pull request by ChiefAlexander**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by carrodher** [comment]
If the goal of this PR is fully deprecating the chart, there are other pending steps needed, you can take a look at the [PROCESSES](https://github.com/helm/charts/blob/master/PROCESSES.md#deprecating-a-chart ) guide (or you can use a previous deprecation PR like https://github.com/helm/charts/pull/21415)

If it is just an informative note but not the definitive deprecation I think it's fine.

File: stable/nginx-ingress/README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# nginx-ingress
+# DEPRECATED - nginx-ingress
```
Comment (davidkarlsen) [inline stable/nginx-ingress/README.md:1]: Please see https://github.com/helm/charts/blob/master/PROCESSES.md for steps to deprecate a chart

**Comment by ChiefAlexander** [comment]
@lachie83 or @taharah could we get a review on this to get it merged in.

**Comment by calvinbui** [comment]
could a migration path be added similar to https://github.com/helm/charts/pull/21415/files#diff-179171755c7eed95fc128b80a12cb527 ?

either in this repo or the ingress-nginx repo with a link to it

**Comment by scottrigby** [comment]
Cross referencing https://github.com/kubernetes/ingress-nginx/pull/6038

**Review by scottrigby — APPROVED** [review]
/lgtm

**Pull request merged**